dask/dataframe/io/csv.py

```diff
@@ -507,6 +507,8 @@ def to_csv(df, filename, name_function=None, compression=None, compute=True,
     header : boolean or list of string, default True
         Write out column names. If a list of string is given it is assumed
         to be aliases for the column names
+    header_first_chunk : boolean, default False
```

Comment:
Not sure about this name, as it could be interpreted as "write the header on the first chunk" or "write the header on the first chunk *only*`. So maybe `header_first_chunk_only`?

Reply:
And maybe `partition` instead of `chunk`.

Reply:
"chunk" is more in line with the terminology used in dask and pandas - the "chunksize" argument passed to pandas and the to_csv_chunk function internally in dask